airbyte-workers/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java

```diff
@@ -163,6 +199,14 @@ public void close() throws Exception {
     }
   }
 
+  @Override
+  public Stream<AirbyteTraceMessage> getTraceMessages() {
+    if (airbyteMessagesByType != null) {
+      return airbyteMessagesByType.getOrDefault(Type.TRACE, new ArrayList<>()).stream().map(AirbyteMessage::getTrace);
```

Comment:
Instead of `new ArrayList<>()).stream().map(AirbyteMessage::getTrace` it should be `Stream.<AirbyteTraceMessage>empty()`, the map is confusing

Reply:
`airbyteMessagesByType.get(Type.TRACE)` is a `List<AirbyteMessage>` so we need the map to get `Stream<AirbyteTraceMessage>`.

I've added another check in the `if` on `airbyteMessagesByType.get(Type.TRACE)` being null so we don't need the `new ArrayList<>()` default.